.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "eslint-config-edx"
+}
```

Comment:
Yeah, I figure this PR will establish the high-water mark for violations and then we can draw it down as we come across things we think should be exceptions in this repo. Cale and I talked about this a bit [here](https://github.com/edx/edx-platform/pull/13081#discussion_r72491949)